weatherman.py

```diff
@@ -0,0 +1,109 @@
+"""
+Weatherman
+This module generates yearly reports on weather data collected from  Dec 1996 - May 2011.
+Report no. 1 gives the yearly max/min temperature and max/min humidity.
+Report no. 2 gives the hottest day of the year and the temperature on that day.
+"""
+
+import os
+import sys
+
+import argparse
+
+
+max_temp_ = {}
+max_temp_date = {}
+min_temp_ = {}
+max_humidity= {}
+min_humidity= {}
+
+parser = argparse.ArgumentParser(prog = 'weatherman',
+								 usage = '%(prog)s [report#] [data_dir]',
+								 description = 'Generate weather reports')
+parser.add_argument('-d', '--data_dir', type = str, choices = ['./weatherdata/'],
+					help = 'Directory containing weather data files', required = True)
+parser.add_argument('-r', '--report_number', type = int, choices = [1, 2],
+					help = '1 for Annual Max/Min Temperature, 2 for Hottest day of each year',
+					required = True)
+try:
+	args = parser.parse_args()
+except:
+    parser.print_help()
+    sys.exit(0)
+
+"""
+Used for printing yearly reports.
```

Comment:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
View the link to learn how to write doc strings.

Reply:
Viewed and edited according, please see if an improvement can still be made.

Reply:
Viewed and edited according, please see if an improvement can still be made.